### PR TITLE
add trackVerifiedToken()

### PR DESCRIPTION
**Pull request by nickpatrick**
- exposes `Radar.trackVerifiedToken()`
- adds a new demo page

image

### CODE REVIEW

File: demo/views/track-verified-token.hbs [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  </div>
+
+  <div id="response-details" style="display: none;">
+    <h6 class="mt-4">Token</h6>
+    <div class="mt-2">
+      <pre style="white-space: pre-wrap"><code id="token" class="code-sample language-json"></code><pre>
+    </div>
+  </div>
+
+  <div class="mt-4">
```
Comment (kochis) [inline demo/views/track-verified-token.hbs:30]: We can probably omit the code samples on these. These should probably live in the docs (I only added them as a temporary solution until we go the UI kits in the docs).

File: src/api/verify.ts [hunk 2]
```diff
@@ -49,27 +50,39 @@ class VerifyAPI {
...
+      const trackTokenRes = {
+        token,
+      } as RadarTrackTokenResponse;
+  
+      if (options.debug) {
+        trackTokenRes.response = response;
+      }
+  
+      return trackTokenRes;
+    } else {
```
Comment (kochis) [inline src/api/verify.ts:73]: Technically don't need this `else` since it returns above.

**Review by kochis — DISMISSED** [review]

File: demo/views/track-verified-token.hbs (hunk 1 above)
Comment (nickpatrick) [reply demo/views/track-verified-token.hbs:30]: I've actually found it helpful to be able to show the code sample when using this page in demos (which I'm doing a lot), but we can always remove later

File: src/api/verify.ts (hunk 2 above)
Comment (nickpatrick) [reply src/api/verify.ts]: Updated

**Pull request merged**